utils/consts.go

```diff
@@ -2195,6 +2195,7 @@ const (
 	OptsContext                = "*context"
 	Subsys                     = "*subsys"
 	MetaMethod                 = "*reqMethod"
+	OptsAccountsUsage          = "*accountsUsage"
```

Comment:
In order to be consistent, you need to keep also account options (as all the other) prefixed with 3-4 chars instead of full accounts or rates.